chart/templates/_helpers.tpl

```diff
@@ -0,0 +1,63 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
@MaximeStOnge Will the references in this file be changed when you run helm commands?